numba/inline_closurecall.py

```diff
@@ -1,11 +1,18 @@
 from numba import config, ir, ir_utils, utils
 import types
 
-from numba.ir_utils import (mk_unique_var, next_label, add_offset_to_labels, 
-    replace_vars, remove_dels, remove_dead, rename_labels)
- 
+from numba.ir_utils import (mk_unique_var, next_label, add_offset_to_labels,
+    replace_vars, remove_dels, remove_dead, rename_labels, find_topo_order, merge_adjacent_blocks)
+from numba.analysis import compute_cfg_from_blocks, find_top_level_loops
+from numba.transforms import _extract_loop_lifting_candidates, _loop_lift_get_candidate_infos
+from numba.targets.rangeobj import range_iter_len
+import numpy
```

Comment:
I looked at it again, `numpy` is only imported for its module name, not for accessing numpy functions. Do you still prefer using `import numpy as np` in this case?